tpgdata/vehicules/vehicules.php

```diff
@@ -390,16 +390,16 @@ public function __construct(string $id) {
             $this->icon = 'c2';
             $this->img = 'hybride';
             $this->img_author = 'TramBusAl';
-            $this->img_link = 'https://twitter.com/trambusalOff';
+            $this->img_link = 'https://flickr.com/photos/trambusal/';
 
         } elseif (2003 <= $id && $id <= 2010) { // Sprinter
 
             $this->type = "Minibus";
             $this->name = "Mercedes-Benz Sprinter City 65";
-            $this->icon = 'notfound';
+            $this->icon = 'altas';
```

Comment:
This icon seems to be missing, did you mean to add a new one?

Reply:
oops, my fault forgot to add it to the repo 🤦